### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by trasher**
Translate index page
Begin to translate prerequisites page

See build: http://glpi-install.readthedocs.io/en/feature-translation/

### CODE REVIEW

**Comment by trasher** [comment]
Contents have been translated. I have to update various screenshot (they'll all be in english).

File: source/command-line.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
+Command line tools
+==================
+
+Since GLPI 9.2.2, command line tools are provided as supported scripts and are availaible from the ``scripts`` directory of the archive. On previous versions, thos scripts were present in the ``tools`` directory that is not official and therefore not in the release archive.
```
Comment (orthagh) [inline source/command-line.rst]: thos -> those

File: source/command-line.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+   Do not forget to backup your database before any update try!
+
+Possible options for this command are:
+
+ * ``--lang`` language code to use (`fr_FR` as example). Will be set to `en_GB` per default,
+ * ``--help`` displays command help.
+ * ``--config-dir`` set configuration file path to use,
+ * ``--force`` force update, usefull when GLPI version does not change (mainly when working on it ;)),
+ * ``--dev`` required to use a development version.i Ise with cautions...
```
Comment (orthagh) [inline source/command-line.rst]: "i Ise" ???

maybe "Use it"

File: source/index.rst [hunk 3]
```diff
@@ -3,17 +3,18 @@
...
-Installation de GLPI
-====================
+GLPI installation
+=================
 
-Cette documentation présente l'installation et la prise en main globale de l'application `GLPI (Gestion Libre de Parc Informatique) <http://glpi-project.org/>`_ !
+This documentation presents `GLPI <http://glpi-project.org>`_ installation instructions.
 
-Solution libre et open-source de gestion de parc informatique et de helpdesk, GLPI est une application accessible via un navigateur web conçue pour gérer l'ensemble de vos problématiques de gestion de parc informatique : de la gestion de l'inventaire des composantes matérielles et logicielles d'un parc informatique à la gestion de l'assistance aux utilisateurs.
+:abbr:`GLPI (Gestion Libre de Parc Informatique)` is a free (as in beer!) asset and helpdesk management solution accessible from a web browser built to manage all you asset management issues, from hardware components and software inventories management to user hdelpdesk management.
```
Comment (orthagh) [inline source/index.rst]: not sure about "as in beer" sentence, first, it should be "as a beer", but the principle of foss, is, i think, exactly the opposite: free as speech not as a beer, see: https://www.gnu.org/philosophy/free-sw.en.html

File: source/install/index.rst [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+#. Install :)
+
+Choose a version
+----------------
+
+.. note::
+
+   It is hightly recommended you choose the latest stable release for a production usage.
+
+GLPI follows a semantic versionning scheme, on 3 digits. The first one is the majo release, the second the minor and the third the fix release.
```
Comment (orthagh) [inline source/install/index.rst]: majo -> major

File: source/install/index.rst [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+Fixes releases will only fix reported issues without adding anything new.
+
+Download
+--------
+
+.. warning::
+
+   On GitHub, there is always two archives named *Source code* which should not be used.
+
+Go to the *download* section of `the GLPI website <http://glpi-project.org>`_ or get archive directly from `Github release <https://github.com/glpi-project/glpi/releases>`_.
```
Comment (orthagh) [inline source/install/index.rst]: i would add: and choose the ``xx.tar.bz`` file

File: source/install/index.rst [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+
+Installation
+------------
+
+.. note::
+
+   Packages may be availaible from your distribution (Red Hat, CentOS, Fedora, Ubuntu, ...) that you should use with your standard packages manager as usual.
```
Comment (orthagh) [inline source/install/index.rst]: availaible -> available

File: source/install/index.rst [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+
+GLPI installation itself is composed of three steps:
+
+#. Uncompress archive in your website;
+#. Give your webserver write access to ``files`` and ``config`` directories;
+#. :doc:`launch installation wizard <wizard>` (or use :ref:`command line installation script <cdline_install>`).
+
+Once three steps has been completed the app is ready to be used.
+
+Files and directories emplacements
```
Comment (orthagh) [inline source/install/index.rst]: emplacements, better use "locations", no ?

File: source/install/index.rst [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+
+There are a few configuration directives you may use to achieve that (directives that are used in provided downstream packages):
+
+* ``GLPI_CONFIG_DIR``: set path to the configuration directory;
+* ``GLPI_VAR_DIR`` : set path to the ``files`` directory;
+* ``GLPI_LOG_DIR`` : set path to logs files.
+
+.. note::
+
+   There ara many other configuration directives availaible, the ones we talked about are the main ones to take into account for a more secure installation.
```
Comment (orthagh) [inline source/install/index.rst]: ara -> are
main ones -> main

File: source/install/index.rst [hunk 9]
```diff
@@ -0,0 +1,126 @@
...
+
+Directories choice is entirely up to you; the following example will follow the `FHS <http://www.pathname.com/fhs/>`_ recommandations.
+
+Our GLPI instance will be installed in ``/var/www/glpi`` directory, a specific virtual host in the web server configuration will reflect this path.
+
+GLPI configuration will be stored in ``/etc/glpi``, just copy the ``config`` directory to this place. GLPI requires read rights on this directory to work; and write rights during the installation process.
+
+GLPI data will be stored in ``/var/lib/glpi``, just copy the ``files`` directory to this place. GLPI requires read and write rights on this directory.
+
+GLPI logs files will be stored in ``/var/log/glpi``, there is nothing to copye here, just create the directory. GLPI requires read and write access on this directory.
```
Comment (orthagh) [inline source/install/index.rst]: copye -> copy

File: source/install/wizard.rst [hunk 10]
```diff
@@ -0,0 +1,137 @@
...
+
+.. image:: images/select_language.png
+   :alt: Choose lang
+   :align: center
+   :scale: 50%
+
+License
+-------
+
+Usage of GLPI is subject to GNU license approval. One licensing terms read an accepted, just validate the form.
```
Comment (orthagh) [inline source/install/wizard.rst]: One -> Once

File: source/install/wizard.rst [hunk 11]
```diff
@@ -0,0 +1,137 @@
...
+
+.. image:: images/license_agreement.png
+   :alt: Licensing terms
+   :align: center
+   :scale: 50%
+
+If you do not agreee with licensing terms, it is not possible to continue installation process.
```
Comment (orthagh) [inline source/install/wizard.rst]: agreee -> agree

File: source/install/wizard.rst [hunk 12]
```diff
@@ -0,0 +1,137 @@
...
+
+Install / Update
+----------------
+
+This screen allows to choose beetween a fresh GLPI installation or an update.
```
Comment (orthagh) [inline source/install/wizard.rst]: beetween -> between

File: source/install/wizard.rst [hunk 13]
```diff
@@ -0,0 +1,137 @@
...
+   :alt: Install or update
+   :align: center
+   :scale: 50%
+
+Click on install.
+
+Environment checks
+^^^^^^^^^^^^^^^^^^
+
+This step will check if prerequisites are met. If thery're not, it is not possible to continue and an explicit error message will tell you about what is wonrg and what to do before trying again.
```
Comment (orthagh) [inline source/install/wizard.rst]: wonrg -> wrong

File: source/install/wizard.rst [hunk 14]
```diff
@@ -0,0 +1,137 @@
...
+   :scale: 50%
+
+* *MySQL server*: enter the path to your MySQL server, `localhost` or `mysql.domaine.tld` as example;
+* *MySQL user*: enter user name that is allowed to connect to the Database;
+* *MySQL password*: enter user's password.
+
+Once all fields are properly filled, validate the form.
+
+A first database connection is then established. If parameters are invalid, an error message will be displayed, and you'll have to fix parameters and try again.
+nouveau.
```
Comment (orthagh) [inline source/install/wizard.rst]: nouveau ?

File: source/install/wizard.rst [hunk 15]
```diff
@@ -0,0 +1,137 @@
...
+   Choose *Create a new database*, enter the database name in the relevant field and then validate to create the base.
+
+   .. warning::
+
+      SQL user must be able to create new database for this option to work.
+
+Database initialization
+^^^^^^^^^^^^^^^^^^^^^^^
+
+This steps initializes the database with default values.
```
Comment (orthagh) [inline source/install/wizard.rst]: steps -> step

File: source/install/wizard.rst [hunk 16]
```diff
@@ -0,0 +1,137 @@
...
+
+.. image:: images/db-ok.png
+   :alt: Database initialization
+   :align: center
+   :scale: 50%
+
+If there is any error; py attention to the displayed informations.
```
Comment (orthagh) [inline source/install/wizard.rst]: py -> pay

File: source/install/wizard.rst [hunk 17]
```diff
@@ -0,0 +1,137 @@
...
+
+End of installation
+^^^^^^^^^^^^^^^^^^^
+
+This step presents a summary of the installation and give created users list. Please pay attention to those informations and validate to got to the app.
```
Comment (orthagh) [inline source/install/wizard.rst]: got -> go

File: source/command-line.rst [hunk 18]
```diff
@@ -0,0 +1,49 @@
+Command line tools
+==================
+
+Since GLPI 9.2.2, command line tools are provided as supported scripts and are availaible from the ``scripts`` directory of the archive. On previous versions, those scripts were present in the ``tools`` directory that is not official and therefore not in the release archive.
```
Comment (orthagh) [inline source/command-line.rst]: availaible - > available

File: source/index.rst [hunk 19]
```diff
@@ -3,17 +3,18 @@
...
-Installation de GLPI
-====================
+GLPI installation
+=================
 
-Cette documentation présente l'installation et la prise en main globale de l'application `GLPI (Gestion Libre de Parc Informatique) <http://glpi-project.org/>`_ !
+This documentation presents `GLPI <http://glpi-project.org>`_ installation instructions.
 
-Solution libre et open-source de gestion de parc informatique et de helpdesk, GLPI est une application accessible via un navigateur web conçue pour gérer l'ensemble de vos problématiques de gestion de parc informatique : de la gestion de l'inventaire des composantes matérielles et logicielles d'un parc informatique à la gestion de l'assistance aux utilisateurs.
+:abbr:`GLPI (Gestion Libre de Parc Informatique)` is a free (as in "free speech" not as in "free beer"!) asset and helpdesk management solution accessible from a web browser built to manage all you asset management issues, from hardware components and software inventories management to user hdelpdesk management.
```
Comment (orthagh) [inline source/index.rst]: hdelpdesk -> helpdesk

File: source/install/index.rst [hunk 20]
```diff
@@ -0,0 +1,126 @@
+Install GLPI
+============
+
+Preceed as follow:
```
Comment (orthagh) [inline source/install/index.rst]: Preceed -> Proceed

File: source/install/wizard.rst [hunk 21]
```diff
@@ -0,0 +1,148 @@
...
+
+.. image:: images/select_language.png
+   :alt: Choose lang
+   :align: center
+   :scale: 50%
+
+License
+-------
+
+Usage of GLPI is subject to GNU license approval. Once licensing terms read an accepted, just validate the form.
```
Comment (orthagh) [inline source/install/wizard.rst]: an -> and

File: source/install/wizard.rst [hunk 22]
```diff
@@ -0,0 +1,148 @@
...
+^^^^^^^^^^^^^^^^^^
+
+This step will check if prerequisites are met. If thery're not, it is not possible to continue and an explicit error message will tell you about what is wrong and what to do before trying again.
+
+.. image:: images/setup.png
+   :alt: Check prerequisites
+   :align: center
+   :scale: 50%
+
+Some prerequisites are optionals, it will be possible to continue installation event if thery're not met.
```
Comment (orthagh) [inline source/install/wizard.rst]: optionals -> optional ? (not sure about this one)

File: source/prerequisites.rst [hunk 23]
```diff
@@ -0,0 +1,83 @@
...
+* ``mysqli``: to connect and query the database liaison avec la base de données ;
+* ``session``: to get user sessions support;
+* Zlib: to get backup and restore database functions.
+
+Optional extensions
+^^^^^^^^^^^^^^^^^^^
+
+.. note::
+
+   Even if those extensions are not mandatory, we advice you to install them anyways.
```
Comment (orthagh) [inline source/prerequisites.rst]: advice -> advise

File: source/install/wizard.rst (hunk 22 above)
Comment (trasher) [reply source/install/wizard.rst]: I guess there is a 's' since we're talking about several prerequisites

**Pull request merged**